### DISCUSSION TITLE
When I click on walletConnect, how do I display only walletConnect pop-ups?

### DISCUSSION THREAD

**Discussion by Garrett-Wind** [rainbow-me/rainbowkit#1724] [Feedback]
![1706078480545](https://github.com/rainbow-me/rainbowkit/assets/15326862/eff01185-7476-4511-82d1-1f8115bff5c5)
![1706078407693](https://github.com/rainbow-me/rainbowkit/assets/15326862/433b1e6c-db26-4eba-ab40-6fd7b981f888)

I wrote showQrModal: true in the options of the walletConnectWallet, why are both displayed at the same time, I want to show only the walletConnect popup

**Comment by 0xmkh** [comment]
@wytweb We do not support "only" wallet connect modal on desktop. We do support it in mobile. If you would like to have "only" wallet connect modal we can have a look on our end and probably expose a prop somewhere to our connectors so this will be possible. 

I don't think there will be a huge UX advantage here. Since we already show a qr code for the user and we have a button to open the official wallet connect modal if the user wishes to open that instead.

**Reply by Garrett-Wind** [reply]
Thank you very much for your reply! 
Because I have an application scenario that requires users to copy the login information and log in through the 6551 wallet on another webpage, some users can not find the place where the copy information will be confused. So I thought if I could just pop up wallet connect modal from the start.
Hope can support this feature in the future, thank you very much ！

**Reply by 0xmkh** [reply]
That make sense. I'll see if i can have something for you during our wagmi v2 and rainbowkit v2 migration. This is fully possible and i think if the qr modal isn't needed and only the wallet connect modal is needed then we'll just make a small change to our connectors. Will keep this open until we've released something on our end 🙏
